todo/src/components/todo.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { todo, todo2 } from '../actions';
+
+const list = props => {
+    
```

Comment:
Remember that even functional components should return some sort of HTML element like a div.